drivers/serial/uart_stm32.c

```diff
@@ -615,6 +630,13 @@ static int uart_stm32_fifo_fill(const struct device *dev,
 {
 	USART_TypeDef *UartInstance = UART_STRUCT(dev);
 	uint8_t num_tx = 0U;
+	int key;
+
+	if (!LL_USART_IsActiveFlag_TXE(UartInstance))
+		return 0;
```

Comment:
Needs brackets even for a one line statement.

Reply:
I had the brackets